heartbeat/look/look_test.go

```diff
@@ -0,0 +1,60 @@
+package look
+
+import (
+	"testing"
+	"time"
+
+	"fmt"
+
+	"github.com/stretchr/testify/assert"
+
+	reason2 "github.com/elastic/beats/heartbeat/reason"
```

Comment:
I would normally rename the variable instead of the package but purely personal preference.